### PR TITLE
Feature/movie cards

### PR DESCRIPTION
**Pull request by Swatzell**
This PR will allow movie cards using the mock data to display on the main page with images of each movies poster.

### CODE REVIEW

File: src/Component/MovieCard/MovieCards.js [hunk 1]
```diff
@@ -1 +1,13 @@
-import './MovieCards.css'
\ No newline at end of file
+import './MovieCards.css'
+
+
+function MovieCards({ movie }){
```
Comment (LISims88) [inline src/Component/MovieCard/MovieCards.js:4]: Are we destructuring movies later on in the project?

**Review by LISims88 — APPROVED** [review]
Code looks clean and easy to follow. Everything make sense.

File: src/Component/MovieCard/MovieCards.js (hunk 1 above)
Comment (LISims88) [reply src/Component/MovieCard/MovieCards.js:4]: Noticed that the Posters need their ratings underneath after I pulled.

Comment (Swatzell) [reply src/Component/MovieCard/MovieCards.js:4]: Good catch! I'll fix that tomorrow. Totally forgot. I was thinking I might wait for API to destructure but I might go ahead and start tomorrow when I fix the ratings if it's not gonna cause any issues.

**Pull request merged**